MANIFEST.in

```diff
@@ -1,3 +1,6 @@
-include curl_cffi/_wrapper.*
-include curl_cffi/ffi/*
-include curl_cffi/include/curl/*
+recursive-include tests/*
+
+include ffi/*
+include include/curl/*
```

Comment:
@yifeikong `include` folder is available in sdist.
Can we make test through ci to install from sdist?

Reply:
I don't think so, for archs on GitHub actions, we would already build bdist wheels, there is no need to test the sdist on these platforms.

I think we should upload the sdist artifacts in `test.yaml` and then down it and test it manually on Raspberry pi and termux.

Reply:
> I think we should upload the sdist artifacts in `test.yaml` and then down it and test it manually on Raspberry pi and termux.

Already done. Can you check the artifact? I don't have Raspberry Pi or Termux available currently.

Reply:
Sorry, I did not see the curl header files in sdist.